Makefile

```diff
@@ -29,3 +29,14 @@ test-notebook-parameters:
 	@echo "Running notebook parameters validation..."
 	pytest tests/test_notebook_parameters.py -v
 	@echo "Notebook parameters test passed :)"
+
+test-notebook-execution:
+	@echo "Running notebook execution tests..."
+	pytest tests/test_notebook_execution.py -v
+	@echo "Notebook execution tests passed :)"
+
+test-notebooks: format-notebooks-check test-notebook-parameters test-notebook-execution
+	@echo "All notebook validations completed successfully (formatting, parameters, execution) :)"
+
+test-all: format-python-check test-notebooks
```

Comment:
Shouldn't everything be here? So all of `format-python format-notebook format-python-check format-notebooks-check test-notebook-parameters test-notebook-execution test-kfp-components test-notebooks test-all`